src/main/webapp/WEB-INF/jsp/help.jsp

```diff
@@ -104,19 +104,21 @@
                 <p><strong>DO NOT EDIT OR DELETE THE ATTENDANCE ASSIGNMENT FROM THE ASSIGNMENTS TAB IN CANVAS.</strong> </p>
                 <br>
                 <h4>Editing: </h4>
-                <p>If you'd like to change the assignment name, the total points, or the weight of any particular
-                attendance status, make all of your changes through the K-State Attendance application. Make the desired
-                changes by creating a new Setup on the Course Setup page then saving the setup for the appropriate
-                section.</p>
+                <p>If you'd like to change the Assignment Name, Total Points, or Attendance Weights for an assignment, you must
+                make all of your changes through the K-State Attendance application. To make the desired changes, create a new
+                Setup on the Course Setup page then save.</p>
                 <br>
                 <h4>Deleting: </h4>
-                <p>Deleting an attendance assignment is done by un-checking the "Convert Attendance to Assignment" checkbox.
-                The dialog pictured below will pop up and answering "Yes" will remove all saved Setups and all attendance
-                assignments. Please do not delete the assignment through the Canvas Assignments tab, instead using this tool.
-                This will <strong>NOT</strong> have any affect on the attendance records you have saved in the course.</p>
+                <p>If you'd like to delete an Attendance Assignment, un-check the "Convert Attendance to Assignment" checkbox. The
+                dialog box (pictured below) will appear. By selecting "Yes", all saved Setups and all Attendance Assignments
+                will be removed. <strong>DO NOT</strong> delete the assignment through the Canvas Assignments tab. Changing or
+                deleting the assignment in the Attendance application will <strong>NOT</strong> have any affect on the attendance
+                records you have saved in the course</p>
                 <br>
-                <p><strong>NOTE:</strong> If you receive an error after recently editing or deleting an assignment, please send us an email including the
-                URL of the error page in the body of the email.</p>
+                <p><strong>NOTE:</strong> If you receive an error after editing or deleting an assignment, send an email to
+                helpdesk@ksu.edu. You will need to copy and paste the URL of the error page in the body of your email.</p>
+                <br>
+                <img src="${context}/img/DeleteConfirmation.png">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)